scripts/release

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+const execSync = require('child_process').execSync;
+const minimist = require('minimist');
+const standardVersion = require('standard-version');
+const chalk = require('chalk');
+const args = minimist(process.argv.slice(2));
+
+const DEFAULT_UPSTREAM_REPO_ORIGIN = 'origin';
+const DEFAULT_UPSTREAM_REPO_BRANCH = 'master';
+const origin = args['repo-origin'] || DEFAULT_UPSTREAM_REPO_ORIGIN;
+const branch = args['repo-branch'] || DEFAULT_UPSTREAM_REPO_BRANCH;
```

Comment:
Shall we allow to change that?
I would personally prefer to use always origin and master
To avoid a weird mistake

Reply:
So `origin` and `master` is always the default. but you can change with `repo-origin` and `repo-branch` options.

Are you referring to something else?

Reply:
I would personally not allow anyone to do a release on something else than master and that is not up to date with GitHub
=> Safe release

Reply:
Ignore that we have the patch.
But at least I would check with the upstream branch

Reply:
> I would personally not allow anyone to do a release on something else than master and that is not up to date with GitHub

So I actually found this useful to have to test this stuff. But again, `origin` and `master` are the default. Also, we'll have release branches from which we want to release from in the future.

> But at least I would check with the upstream branch

Yea we should check that I agree